### DISCUSSION TITLE
How do I decide if I should add myself as one of the definition owners?

### DISCUSSION THREAD

**Discussion by alxtsg** [DefinitelyTyped/DefinitelyTyped#61574] [_Misc]
3 weeks ago I [submitted](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/61266) a pull request for several changes in the definitions of [carbone](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/carbone). The definition owner didn't review and the later the PR finally get reviewed by a DT maintainer.

Looking at the owner's GitHub profile and repositories, I think the definition owner isn't active on GitHub anymore.

I use the module at work and I want to add more enhancements to the definitions.

Should I add myself as one of the definition owners to speedup the review and merge process, instead of waiting for weeks?

**Comment by Dabolus** [comment]
As far as I know, there isn't an official process to become a definitions owner/maintainer. When you edit some typings, simply add your name to the list of authors. Note that more than one name can be present in that field, so you don't need (and you shouldn't) remove the previous author's name. Once you add yourself, you will get the faster review and merge process and you will automatically be asked for a review if someone else opens a PR for that same package definitions. See [this package](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node-telegram-bot-api/index.d.ts) as an example of how to handle multiple authors (it is the first package to which I added some missing definitions).
